charts/rookout-hybrid/README.md

```diff
@@ -23,12 +23,37 @@ helm repo update
 
 ### Installation
 
-[This yaml file](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/examples/nginx_lets_encrypt.yaml) shows a basic example configuration using Nginx + cert-manager + Let's Encrypt.
-
-To use it, modify the contents of the example yaml file to match your configuration, and then run:
+To use it, get token from Rookout UI settings > token , and then run:
 
 ```commandline
-helm upgrade --install rookout rookout/rookout-hybrid -f ingerss_example.yaml
+helm upgrade --install rookout rookout/rookout-hybrid \
+    --set rookout.token="YOU_TOKEN_HERE" \
+    --create-namespace rookout
+```
+
+### Datastore access
+
+This installation will not expose your datastore to your web-browser (Rookout client), ingress or other method you are using should expose the datastore service to your web-browser.
+
+<img src="https://docs.rookout.com/assets/images/datastore_diagram-628f79ca44d7af9c355c6f0f7d821712.png" width="900">
+
+you can expose datastore using ingress or chaning the service type of it to LoadBalancer or NodePort as preffered. for reference checkout [Values.yaml](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/values.yaml)
+
+Here is naive example using ingress, nginx and let's encrypt issuer.
+[nginx_lets_encrypt.yaml](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/example/nginx_lets_encrypt.yaml)
+
+### Rooks configuration
+
+If the rooks (agents) in same cluster as Rookout's controller they can communicate using internal k8s DNS server, the address will be `CONTROLLER_SERVICE_NAME.NAMESPACE.svc.cluster.local` the default domain with the above command would be `controller-rookout-rookout-hybrid.rookout.svc.cluster.local`. If the rooks not in same cluster, expose the controller's service as you did for the datastore.
```

Comment:
Change to
"If the agents are in same cluster as Rookout's controller, they can communicate using the internal k8s DNS server. The address will be `CONTROLLER_SERVICE_NAME.NAMESPACE.svc.cluster.local` and the default domain will be `controller-rookout-rookout-hybrid.rookout.svc.cluster.local`. If the agents are not in same cluster, you must expose the controller's service as you did for the datastore."